### PR TITLE
Cheryl updates

### PR DESCRIPTION
**Pull request by moserc**
* login/logout fixed
* reservation form element generated inside of 'item detail' determined by login status
  > STILL TO DO: db / back end needs to generate and return a confirmation number
* review form - NOTE: endpoint url may need editing after it is created
* search bar started, endpoint pending

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@ <h1>
                     Welcome to Go Outside Co.
                     <br>Equipment Reservations
                 </h1>
-                <nav>reservations home || <a href="login.html">login</a></nav>
+                <nav>reservations home || <a href="login.html" id="log">log in</a></nav>
```
Comment (codingwithcookie) [inline public/index.html:27]: log is a common name would `loginLink` be a better id?

File: public/login.html [hunk 2]
```diff
@@ -1,50 +1,50 @@
...
+                </h1>
+                <nav><a href="index.html">reservations home</a> || <a href="#" id="log">log in</a></nav>
+            </hgroup>
+        </section>
+
+        <section id="content">
+            <section id = "result">
+                <form id="user_login" action="/api/user/login" method="post">
+                    <div>
+                        <label for="email">username: </label>
```
Comment (codingwithcookie) [inline public/login.html:35]: If we are using email/password, should this be renamed to email and leave out the concept of username?

File: public/register.html [hunk 3]
```diff
@@ -1,50 +1,54 @@
...
+                    <br>Equipment Reservations
+                </h1>
+                <nav><a href="index.html">reservations home</a> || <a href="login.html" id="log">log in</a></nav>
+            </hgroup>
+        </section>
+
+        <section id="content">
+            <section id = "result">
+                <form id="create_user" action="/api/user/register" method="post">
+                    <div>
```
Comment (codingwithcookie) [inline public/register.html:34]: If we are using email/password can this be removed?

File: public/transactions.html [hunk 4]
```diff
@@ -1,46 +1,46 @@
...
+    </head>
+
+    <body>
+        <section id = "top">
+            <hgroup>
+                <h1>
+                    Welcome to Go Outside Co.
+                    <br>Equipment Reservations
+                </h1>
+                <nav><a href="index.html">reservations home</a> || <a href='#' id='log'>log out</a></nav>
```
Comment (codingwithcookie) [inline public/transactions.html:27]: Should this id be named log?

File: public/register.html (hunk 3 above)
Comment (moserc) [reply public/register.html:34]: Remove the ability to add new users to the database?

File: public/index.html (hunk 1 above)
Comment (moserc) [reply public/index.html]: changed all instances of id='log' to id='user_status'

File: public/login.html (hunk 2 above)
Comment (moserc) [reply public/login.html]: done

File: public/register.html (hunk 3 above)
Comment (codingwithcookie) [reply public/register.html:34]: The part where we ask for a username.

**Review by codingwithcookie — APPROVED** [review]

**Pull request merged**